tests/test_top_level.py

```diff
@@ -0,0 +1,20 @@
+import pyhf
```

Comment:
this should be in the `test_api` instead of a separate python test here. "top level" is vague.